rq/worker_registration.py

```diff
@@ -1,8 +1,10 @@
 from .compat import as_text
 
+from rq.utils import split_list
 
 WORKERS_BY_QUEUE_KEY = 'rq:workers:%s'
 REDIS_WORKER_KEYS = 'rq:workers'
+MAX_REMOVABLE_KEYS = 1000
```

Comment:
Mind renaming this to `MAX_KEYS`? We may reuse this in other places.

Reply:
Done